src/components/LineChartComponent.tsx

```diff
@@ -1,20 +1,43 @@
 import * as React from 'react';
 import { LineChart } from '@mui/x-charts/LineChart';
 import { useTheme } from '@mui/material/styles';
+import Card from '@mui/material/Card/Card';
+import CardContent from '@mui/material/CardContent/CardContent';
+import Typography from '@mui/material/Typography';
 
-export default function LineChartComponent() {
+export type LineChartData = {
+  id: number
+  value: number
+  label: string
+}
+
+export interface LineChartComponentProps{
+  xAxis: [{}] //e.g. xAxis={[{data: [1, 2, 3, 5, 8, 10]}]}
+  yAxis: number[]
```

Comment:
In contrast, `yAxis` probably has to be reimplemented to support multiple lines display, so that all fund data can be displayed on the same chart. The data type might look sth like below, and maybe we got to feed in `series` instead of just one `yAxis` component

```
series: {data: number[], color: "string"}[]
```